international-journal-of-learner-corpus-research.csl

```diff
@@ -0,0 +1,1514 @@
+          </group>
+          <group prefix="(" suffix=")" delimiter=" ">
+            <text variable="authority"/>
+            <choose>
+              <if variable="container-title" match="any">
+                <!--Only print year for cases published in reporters-->
+                <date variable="issued" form="numeric" date-parts="year"/>
+              </if>
+              <else>
+                <date variable="issued" form="text"/>
+              </else>
+            </choose>
+          </group>
+        </group>
+      </if>
+      <else-if type="bill legislation" match="any">
+        <group prefix=", " delimiter=" ">
+          <group delimiter=", ">
+            <choose>
+              <if variable="number">
+                <!--There's a public law number-->
+                <text variable="number" prefix="Pub. L. No. "/>
+                <group delimiter=" ">
+                  <!--change to label variable="section" as that becomes available -->
+                  <text term="section" form="symbol"/>
+                  <text variable="section"/>
+                </group>
+                <group delimiter=" ">
+                  <text variable="volume"/>
+                  <text variable="container-title"/>
+                  <text variable="page-first"/>
+                </group>
+              </if>
+              <else>
+                <group delimiter=" ">
+                  <text variable="volume"/>
+                  <text variable="container-title"/>
+                  <!--change to label variable="section" as that becomes available -->
+                  <text term="section" form="symbol"/>
+                  <text variable="section"/>
+                </group>
+              </else>
+            </choose>
+          </group>
+          <date variable="issued" prefix="(" suffix=")">
+            <date-part name="year"/>
+          </date>
+        </group>
+      </else-if>
+      <else-if type="treaty">
+        <group delimiter=" ">
+          <number variable="volume"/>
+          <text variable="container-title"/>
+          <text variable="page"/>
+        </group>
+      </else-if>
+    </choose>
+  </macro>
+  <!-- As per the IJLCR stylesheet, page numbers are not proceeded by p. or pp. in in-text citations. The following macro was therefore added. -->
+  <macro name="point-locators">
+    <group>
+      <choose>
+        <if locator="page" match="none">
+          <label variable="locator" form="short" suffix=" "/>
+        </if>
+      </choose>
+      <text variable="locator"/>
+    </group>
+  </macro>
+  <!-- The IJLCR stylesheet actually states: "When citing or quoting a publication with three or more authors for the first time in the text, please list all authors’ names. For every consecutive mention of this reference, please give only the first author plus “et al." As far as I know, this cannot be achieved in Zotero. Et al. is therefore always printed when there are 3 or more authors -->
```

Comment:
> <!-- The IJLCR stylesheet actually states: "When citing or quoting a publication with three or more authors for the first time in the text, please list all authors’ names. For every consecutive mention of this reference, please give only the first author plus “et al." As far as I know, this cannot be achieved in Zotero. Et al. is therefore always printed when there are 3 or more authors -->

@elenlefoll, this can be achieved in CSL through use of a `<choose/>` element that test for "position" of the in-text citation, e.g. with `position=”first”` or `position=”subsequent”`. See https://docs.citationstyles.org/en/1.0.1/specification.html#choose. Do you feel comfortable enough making this edit yourself? I didn't check the rest of the XML comments to see if you documented any other issues you ran into. Please let us know if there are.

Reply:
No choose necessary. These are standard et -al-subsequent settings, already used in APA 6, no?

Reply:
Great to hear that this is possible @rmzelle! 

@adam3smith, I've just tested it again and the stylesheet currently does not do this automatically even though I did not make any changes to this section of the original APA 6 template.

How do you recommend I proceed?